torchao/quantization/pt2e/_affine_quantization.py

```diff
@@ -113,7 +113,8 @@ def _get_reduction_params(block_size, input_size):
           shape_for_reduction: (3, 3, 5, 2, 10)
           reduction_dim: [0, 1, 3, 4]
     """
-    assert len(block_size) == len(input_size)
+    assert block_size == [-1] or len(block_size) == len(input_size)
+    block_size = [-1] * len(input_size) if block_size == [-1] else block_size
```

Comment:
is this correct? I think here we need to fill in real dimensions

Reply:
Seems both are working. I will change it to real dimensions. Thanks.